common/fs/src/cache/mod.rs

```diff
@@ -799,6 +804,30 @@ impl FileSystem {
                 .metadata()
                 .map(|m| [Span::new(0, m.len()).unwrap()].iter().collect())
                 .unwrap_or_default(),
+            Lookback::Tail => {
+                let mut should_lookback = false;
+                let file_len = path.metadata().map(|m| m.len()).unwrap_or(0);
+
+                if file_len > TAIL_WARN_THRESHOLD_B {
+                    log::warn!("lookback ocurred on larger file {:?}", path);
```

Comment:
also print the threshold?

Reply:
i'm not sure it's necessary - as it's not configurable - could just be extra noise - and is just a heads up to the user that we are doing a lookback on a not insignificant file.